src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -54,4 +60,43 @@ public void confirmCall(@RequestBody ConfirmResponse data)
       log.error("/v0.5/links/link/confirm : " + data.getError().getMessage());
     }
   }
+
+  // --------------------HIP Linking--------------------//
```

Comment:
Lets remove this kind of comment and add javadoc instead explaining what each method is supposed to do

Reply:
Added javaDoc for all of the methods.